crates/rooch-types/src/chain_id.rs

```diff
@@ -0,0 +1,46 @@
+// Copyright (c) RoochNetwork
+// SPDX-License-Identifier: Apache-2.0
+
+use std::fmt::Display;
+
+#[cfg(any(test, feature = "fuzzing"))]
+use proptest_derive::Arbitrary;
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Copy, Clone, PartialEq, Eq, Hash, Serialize, Deserialize)]
+#[cfg_attr(any(test, feature = "fuzzing"), derive(Arbitrary))]
+#[repr(u64)]
+pub enum ChainID {
+    Dev = 20230103,
+    Test = 20230102,
+    Main = 20230101,
```

Comment:
Or directly represent by 1,2,3, it will be easier.
```
Dev = 3,
Test = 2,
Main = 1
```

Reply:
Rooch's ChainID must not conflict with Ethereum or Ethereum fork chain id.